src/hooks/useBalanceForPosition.tsx

```diff
@@ -22,20 +22,16 @@ export const useBalanceForPosition = (positionId: string) => {
   const balanceData = {
     balanceERC1155: new BigNumber(0),
     balanceERC20: new BigNumber(0),
-    collateralTokenAddress: '',
-    wrappedTokenAddress: '',
```

Comment:
This was creating the problem, If an user was offline, the hook didn't return the collateral information to display in the details